cli/pcluster/commands.py

```diff
@@ -133,12 +133,12 @@ def create(args):  # noqa: C901 FIXME!!!
         LOGGER.debug("StackId: %s", stack.get("StackId"))
 
         if not args.nowait:
-            utils.verify_stack_creation(cfn, stack_name)
+            utils.verify_stack_creation(stack_name, cfn)
             LOGGER.info("")
-            result_stack = cfn.describe_stacks(StackName=stack_name).get("Stacks")[0]
+            result_stack = utils.get_stack(stack_name, cfn)
             _print_stack_outputs(result_stack)
         else:
-            stack_status = cfn.describe_stacks(StackName=stack_name).get("Stacks")[0].get("StackStatus")
+            stack_status = utils.get_stack(stack_name, cfn).get("StackStatus")
```

Comment:
same for `get_stack` function

Reply:
If you mean: rename cfn to cfn_client --> done.